README.md

```diff
@@ -1 +1,37 @@
-### TODO
+# TzSafe
+
+TzSafe is a website to interact with multi-signatures wallets. The goal is to have a UI layer above the LIGO contracts that you can find [here](https://github.com/marigold-dev/tzsafe).
+
+## How to develop ?
+
+First, clone the repository:
+
+```bash
+git clone https://github.com/marigold-dev/Multisig-ui/
```

Comment:
If we change the name of this repo to `tzsafe-ui`, will it break CD? 
(I think CI is fine.)

Reply:
No it won't. The important part is the image name that we build in the CI I think

Reply:
```suggestion
git clone https://github.com/marigold-dev/tzsafe-ui/
```
I renamed the repo to `tzsafe-ui`.  The rest of doc looks good.